yarn.lock

```diff
@@ -14,7 +14,7 @@
   "resolved" "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.15.0.tgz"
   "version" "7.15.0"
 
-"@babel/core@^7.0.0", "@babel/core@^7.0.0-0", "@babel/core@^7.12.0", "@babel/core@^7.13.0", "@babel/core@^7.15.8", "@babel/core@^7.4.0-0":
+"@babel/core@^7.15.8":
```

Comment:
Is there a reason why this changed?

Reply:
Not sure what the original Node version here was.  I used Node 16 and did a fresh install.

Reply:
We use Node 14 and are switching to 18. Let's simply not change files like this unless we actually changed dependencies IMO